project_timesheet_time_control/__openerp__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antonio Espinosa
```

Comment:
No, that depends on the work contract itself. At Tecnativa, we don't remove the copyright from the coder.